plugins/aws_ecr/README.md

```diff
@@ -0,0 +1,19 @@
+# AWS ECR Plugin
+
+This plugin allows integration between Samson and
+[AWS ECR](http://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html),
+which is the Docker registry service managed by AWS.
+
+## Overview
+
+AWS ECR requires you to refresh your registry credentials every 12 hours (approximately).
+Because Samson only supports to set credentials through the environment, this plugin
+runs a callback before every build requesting AWS for the new credentials. Then sets
+these credentials to the environment so the normal docker build process can use them.
+
+To configure this plugin you need to:
+
+* Enable docker in samson (DOCKER_FEATURE=1)
+* Set your ECR registry (DOCKER_REGISTRY=<account>.dkr.ecr.<aws-region>.amazonaws.com)
+* Set your [AWS credentials](http://docs.aws.amazon.com/sdkforruby/api/#Configuration)
+* Set your ECR registry region (AWS_REGION=<aws-region>)
```

Comment:
I went for `AWS_REGION` because it's the default for `aws-sdk` (and I'm pretty sure I saw other places of samson using the default gem env value). Anyway, I can change it to something more specific if you want.